plugins/english/scribblehub.ts

```diff
@@ -149,16 +144,16 @@ class ScribbleHubPlugin implements Plugin.PluginBase {
             return null;
         };
 
-        loadedCheerio('.toc_w').each(function () {
-            const chapterName = loadedCheerio(this).find('.toc_a').text();
-            const releaseDate = loadedCheerio(this).find('.fic_date_pub').text();
-            const chapterUrl = loadedCheerio(this).find('a').attr('href');
+        loadedCheerio('.toc_w').each((i,el) => {
+            const chapterName = loadedCheerio(el).find('.toc_a').text();
+            const releaseDate = loadedCheerio(el).find('.fic_date_pub').text();
+            const chapterUrl = loadedCheerio(el).find('a').attr('href');
 
             if (!chapterUrl) return;
             chapter.push({
                 name: chapterName,
                 releaseTime: parseISODate(releaseDate),
-                url: chapterUrl,
+                path: chapterUrl.replace(this.site, ''),
             });
         });
```

Comment:
I see that this source has pages for chatpers, you should implement Plugin.PagePlugin

Reply:
![image](https://github.com/LNReader/lnreader-sources/assets/26593485/9c57bdfd-bb71-41cb-b0db-6fb2afa74b41)
it uses api to obtain chapters